src/components/Book.js

```diff
@@ -1,30 +1,30 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/Books';
 import InputBook from './Form';
 
-function Book(props) {
-  const { title, author } = props;
+function Book() {
+  const dispatch = useDispatch();
+  const listBooks = useSelector((store) => store.books);
+  console.log(listBooks);
```

Comment:
- [ ] Kindly remove unnecessary console log here. It is not good practice to push them and it kills performance